modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/IgnitePdsCorruptedStoreTest.java

```diff
@@ -154,49 +153,6 @@ private CacheConfiguration cacheConfiguration(String name) {
         return ccfg;
     }
 
-    /**
-     * @throws Exception If test failed.
-     */
-    @Test
-    public void testNodeInvalidatedWhenPersistenceIsCorrupted() throws Exception {
```

Comment:
This test fails with my fix for the following reason: before the fix we always performed binary memory recovery (because we always stopped in the middle of a checkpoint) and we just *happen* to stumble upon a broken page. Now we don't do that, so we don't access the broken page and the node starts successfully

Reply:
Ok, this makes sense